src/Microsoft.AspNetCore.SystemWebAdapters/HttpPostedFileWrapper.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+
+namespace System.Web;
+
+public class HttpPostedFileWrapper : HttpPostedFileBase
```

Comment:
Are we making all of the wrapper types public? Why?

Reply:
This is a public API from .NET Framework: https://learn.microsoft.com/en-us/dotnet/api/system.web.httppostedfilewrapper?view=netframework-4.8.1

It's a convention used for a number of types. Most common is `HttpContext` which has no virtual methods. `HttpContextBase` was an abstraction for it. `HttpContextWrapper` wrapped `HttpContext` to `HttpContextBase`.